sources/packages/backend/apps/api/src/route-controllers/config/config.controller.ts

```diff
@@ -29,6 +29,14 @@ export class ConfigController extends BaseController {
       },
       version: this.configService.apiVersion,
       isFulltimeAllowed: this.configService.isFulltimeAllowed,
+      maximumIdleTimeForWarningStudent:
+        this.configService.maximumIdleTimeForWarningStudent,
+      maximumIdleTimeForWarningSupportingUser:
+        this.configService.maximumIdleTimeForWarningSupportingUser,
+      maximumIdleTimeForWarningInstitution:
+        this.configService.maximumIdleTimeForWarningInstitution,
+      maximumIdleTimeForWarningAest:
+        this.configService.maximumIdleTimeForWarningAest,
```

Comment:
Not a blocker, just a suggestion, these variables can be nested inside an object like the `auth` information for better organization and to avoid repetition.